bundles/org.openhab.binding.solarforecast/README.md

```diff
@@ -59,12 +59,20 @@ See [DateTime](#date-time) section for more information.
 |-----------------|---------|--------------------------------------------------------|-----------------|----------|----------|
 | resourceId      | text    | Resource Id of Solcast rooftop site                    | N/A             | yes      | no       |
 | refreshInterval | integer | Forecast Refresh Interval in minutes                   | 120             | yes      | no       |
```

Comment:
In relation to other comment, could be documented like this:
```suggestion
| refreshInterval | integer | Forecast Refresh Interval in minutes (0 = disable automatic refresh) | 120             | yes      | no       |
```

Reply:
Ups, sure